Casks/v/visualdiffer.rb

```diff
@@ -0,0 +1,19 @@
+cask "visualdiffer" do
+  version "2.1.1"
+  sha256 "814a72a1f8fc83c70216522af034cca464cb6e272e2a29860c700816e726dce7"
+
+  url "https://github.com/visualdiffer/visualdiffer/releases/download/v#{version}/VisualDiffer-#{version}.zip"
+  name "VisualDiffer"
+  desc "Application to visually compare folders and files"
+  homepage "https://github.com/visualdiffer/visualdiffer"
+
+  depends_on macos: ">= :ventura"
```

Comment:
```suggestion
  livecheck do
    url "https://app.visualdiffer.com/appcast.xml"
    strategy :sparkle
  end

  auto_updates true
  depends_on macos: ">= :ventura"
```

It seems that there's an Appcast feed available which we can use to align the `livecheck` with the in-app updater.